packages/manager/apps/telecom/src/app/telecom/pack/xdsl/access/statistics/translations/Messages_fr_FR.json

```diff
@@ -20,5 +20,11 @@
   "xdsl_statistics_decibel_legend": "dB",
   "xdsl_statistics_ping_ms": "{{value}} ms",
   "xdsl_statistics_ping_s": "{{value}} s",
-  "xdsl_statistics_decibel": "{{value}} dB"
-}
\ No newline at end of file
+  "xdsl_statistics_decibel": "{{value}} dB",
+  "xdsl_statistics_errors_title": "Erreurs",
+  "xdsl_statistics_hec_label": "HEC",
+  "xdsl_statistics_fec_label": "FEC",
+  "xdsl_statistics_crc_label": "CRC",
```

Comment:
it is required to translate these? look like they will remain same in all languages. Please move them to a constant file  and use without any translations.

Reply:
Sorry, but the label could be changed in an other time with a real label. So if I use a constant, I will need to break code.